schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -1312,12 +1312,6 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-05-01-preview/Microsoft.Insights.json#/resourceDefinitions/ProactiveDetectionConfigs" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-07-01/Microsoft.ContainerService.json#/resourceDefinitions/containerServices" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-03-31/Microsoft.ContainerService.json#/resourceDefinitions/managedClusters" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-03-20/Microsoft.OperationalInsights.json#/resourceDefinitions/workspaces_savedSearches" },
```

Comment:
There also a few other deploymentTemplate.json (for different versions) that I've updated when followed online doc. But here references removed only from version 2019-04-01. Is it ok?

Reply:
Yeah this is the correct procedure - autogenerated resources are only referenced by the newest 2019-04-01 root schema. We'll be periodically backporting resources into the older root schemas, but this will be a manual process.